README.md

```diff
@@ -108,10 +108,11 @@ The executable was built in Kubuntu 22.04 LTS so there is a chance that it will
 
 #### Install via Flatpak
 
-Thanks to [Massi aka Ergosum](https://github.com/mbugni), Doom Runner can now be installed via [Flatpak](https://flatpak.org/) from this package TODO: link, using the following commands:
```

Comment:
You don't wish to be listed as author? Ok.

Reply:
It's not a problem, the important is that now it's available to download.